### ISSUE TITLE
Atomic species available

### ISSUE THREAD

**Issue by glopez7057** [qsimulate-open/bagel#235]
Hi,
I am trying to run calculations on heavy metals, but putting just a single atom inside the 'molecule' block (at 0 0 0) and running a test calculation that works for say He, I get an error that looks like this (for the example of Cerium):
  ===============================================================
    BAGEL - Freshly leavened quantum chemistry                   
  ===============================================================

  ERROR: EXCEPTION RAISED:  A required keyword is missing from the input:  Ce

Are all elements from the periodic table available within BAGEL? I noticed that this happens for other (heavy) elements as well. I have tried looking around for similar calculations but there are none, and I have not been able to find anything along these lines in the manual/supporting documents.
I appreciate any help/clarification.
Thanks!

**Comment by shiozaki** [comment]
This means that the basis set (or DF basis sets) you chose does not have that element. You could download one from EMSL or create one in the JSON format and specify the full path in the output. Please find some basis sets in src/basis to understand the format.

**Comment by glopez7057** [comment]
Thanks a lot for the quick response. Now I understand the issue, I thought the error was even before the code was getting to that point, but this makes sense now. I will take your recommendations, thanks!

**Comment by glopez7057** [comment]
Not sure if I should open a new issue for this (since this is something very specific to my case), but I have found a decent basis set for the DF and I adapted the format to use it along with the ano-rcc. It seems to be reading it properly, but I get the "Nuclear repulsion" so be zero. Is this prove of an issue, or should I disregard it?
Hopefully it's not an issue to reply here (let me know otherwise). Thanks.
